DatabaseStructures

```diff
@@ -0,0 +1,26 @@
+PHONE
+id
+manufacturer
+model_name
+price
+launch_date
+screen_size
+protection_rating
+processor
+storage
+ram
+main_camera
+selfie_camera
+battery_size
+
+SCORES
+fk_phoneid
+processing_power_scr
+main_camera_scr
+selfie_camera_scr
+storage_scr
+battery_life_scr
+durability_scr
+popularity_scr
```

Comment:
How would these scores be calculated? Would we enter the values manually or would we refer to some 3rd party website/database?

Reply:
This will be an another task.